__tests__/scoreboard.test.js

```diff
@@ -41,7 +42,7 @@ test('one hit', () => {
         totalThrows: 1,
         completedRounds: 0,
         currentRound: new Round(0, 1),
-        hitsPerRound: null
+        hitsPerRound: 3
```

Comment:
Mildly curious why you went with "hitsPerRound" rather than "accuracy" or something? (Ie did you find 1.5 to be more helpful to you than "you hit what you're aiming for 50% of the time"?

Reply:
Not a great explanation but...  My goal is to his one dart per round. I don't know of any standard accuracy metrics for darts, or else I would use that. Also, just using `accuracy` makes it unclear what the units would be.